.github/workflows/post-build.yml

```diff
@@ -198,7 +198,7 @@ jobs:
           MARIADB_ROOT_PASSWORD: wordpress
         ports:
           - 3306:3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
+        options: --health-cmd="healthcheck.sh --su-mysql --connect --innodb_initialized" --health-interval=10s --health-timeout=5s --health-retries=5
```

Comment:
Why `--su-mysql`?

Reply:
Copied from the first example at https://mariadb.com/kb/en/using-healthcheck-sh/#examples, since it seemed to work.